roles/openshift_logging/tasks/install_curator.yaml

```diff
@@ -14,7 +14,7 @@
     component: curator-ops
     logging_component: curator
     deploy_name: "logging-{{component}}"
-    image: "{{image_prefix}}logging-curator:{{image_version}}"
-    es_host: logging-es-ops
-  when: logging_use_ops
+    image: "{{openshift_logging_image_prefix}}logging-curator:{{openshift_logging_image_version}}"
+    openshift_logging_es_host: logging-es-ops
```

Comment:
this should probably stay es_host so that we don't overwrite openshift_logging_es_host for tasks that follow